configs/inference/AliAd/d4/bert4rec_inference_config.json

```diff
@@ -0,0 +1,126 @@
+{
+  "experiment_name": "bert4rec_sanity_check",
+  "dataset": {
+    "type": "scientific",
+    "path_to_data_dir": "./drive/MyDrive/Science/Multidomain RecSys/source/iz-dev/data/AliAd",
+    "dataset_prefix": "d4",
+    "min_sample_len": 5,
+    "max_sample_len": 200,
+    "validation_size": 0.1,
+    "test_size": 0.1,
+    "samplers": {
+      "type": "masked_item_prediction",
+      "mask_prob": 0.4,
+      "negative_sampler": {
+        "type": "random",
+        "sample_size": 99
+      }
+    }
+  },
+  "dataloader": {
+    "train": {
+      "type": "torch",
+      "batch_size": 256,
+      "batch_processor": {
+        "type": "amazon"
+      },
+      "drop_last": true,
+      "shuffle": true
+    },
+    "validation": {
+      "type": "torch",
+      "batch_size": 256,
+      "batch_processor": {
+        "type": "amazon"
+      },
+      "drop_last": false,
+      "shuffle": false
+    }
+  },
+  "model": {
+    "type": "feedforward",
```

Comment:
How does it possibly work for you? These classes were removed a long time ago!!!